handlers.go

```diff
@@ -3052,7 +3052,7 @@ func (s *Server) handleGetViewer(c echo.Context, u *User) error {
 			DealDuration:          dealDuration,
 			MaxStagingWait:        maxStagingZoneLifetime,
 			FileStagingThreshold:  int64(individualDealThreshold),
-			ContentAddingDisabled: s.CM.contentAddingDisabled || u.StorageDisabled,
+			ContentAddingDisabled: s.CM.globalContentAddingDisabled || u.StorageDisabled,
```

Comment:
this is correct for this case

Reply:
> we are checking if the user can still upload content to any box
I think the flag `disable-content-adding` is supposed to include local, at least that's how the code seems to assume it (see https://github.com/application-research/estuary/blob/459a166316370e00aee6388e91501f5668a10d7a/handlers.go#L703-L708)

Reply:
@gmelodie these are the possible outcomes

- if all 3 `s.CM.globalContentAddingDisabled`, `s.CM.localContentAddingDisabled` and `u.StorageDisabled` are `true` => `errorNoUpload`

-  if both `s.CM.globalContentAddingDisabled` and `s.CM.localContentAddingDisabled` are `true` and  `u.StorageDisabled` is `false`  => `errorNoUpload`

- if `s.CM.globalContentAddingDisabled` is `true` and `s.CM.localContentAddingDisabled` are `false` and `u.StorageDisabled` is `true` => `errorNoUpload`

- if `s.CM.globalContentAddingDisabled` is `false` and `s.CM.localContentAddingDisabled` are `true` and `u.StorageDisabled` is `true` => `errorNoUpload`

- if `s.CM.globalContentAddingDisabled` is `true` and `s.CM.localContentAddingDisabled` are `false` and `u.StorageDisabled` is `false` => `uploads to shuttle`

- if `s.CM.globalContentAddingDisabled` is `false` and `s.CM.localContentAddingDisabled` are `true` and `u.StorageDisabled` is `false` => `uploads to estuary`

Reply:
> is supposed to include local, at least that's how the code seems to assume it see:

That was why I said **any** box - local or shuttle